Syntax error in example configuration(s)

The example configuration and documentation have a syntax error regarding `plain_text` for the text formats. The documentation states `plain-text` and it should be `plain_text` otherwise we get the following stack:

```
2023-12-01T09:04:35.170703+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
2023-12-01T09:04:35.170747+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:896)
2023-12-01T09:04:35.171380+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1744)
2023-12-01T09:04:35.171424+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
2023-12-01T09:04:35.171451+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
2023-12-01T09:04:35.172483+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
2023-12-01T09:04:35.172899+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
2023-12-01T09:04:35.172942+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.lang.Thread.run(Thread.java:840)
2023-12-01T09:04:35.173079+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m Caused by: org.springframework.core.serializer.support.SerializationFailedException: Failed to serialize object using DefaultSerializer
2023-12-01T09:04:35.173560+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.springframework.core.serializer.support.SerializingConverter.convert(SerializingConverter.java:64)
2023-12-01T09:04:35.173612+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.springframework.core.serializer.support.SerializingConverter.convert(SerializingConverter.java:33)
2023-12-01T09:04:35.173651+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.springframework.data.redis.serializer.JdkSerializationRedisSerializer.serialize(JdkSerializationRedisSerializer.java:83)
2023-12-01T09:04:35.173876+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011... 136 common frames omitted
2023-12-01T09:04:35.173953+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m Caused by: org.w3c.dom.ls.LSException: class com.sun.org.apache.xerces.internal.dom.TextImpl cannot be cast to class java.lang.String (com.sun.org.apache.xerces.internal.dom.TextImpl is in module java.xml of loader 'bootstrap'; java.lang.String is in module java.base of loader 'bootstrap')
2023-12-01T09:04:35.174541+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.xml/com.sun.org.apache.xml.internal.serializer.dom3.LSSerializerImpl.write(LSSerializerImpl.java:1049)
2023-12-01T09:04:35.174590+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at net.shibboleth.shared.xml.SerializeSupport.writeNode(SerializeSupport.java:134)
2023-12-01T09:04:35.174618+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at net.shibboleth.shared.xml.SerializeSupport.writeNode(SerializeSupport.java:111)
2023-12-01T09:04:35.174684+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at se.swedenconnect.opensaml.common.utils.SerializableOpenSamlObject.writeObject(SerializableOpenSamlObject.java:70)
2023-12-01T09:04:35.175267+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-12-01T09:04:35.175310+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2023-12-01T09:04:35.175347+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-12-01T09:04:35.175844+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.lang.reflect.Method.invoke(Method.java:568)
2023-12-01T09:04:35.175886+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectStreamClass.invokeWriteObject(ObjectStreamClass.java:1070)
2023-12-01T09:04:35.175922+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1516)
2023-12-01T09:04:35.176120+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1438)
2023-12-01T09:04:35.176734+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1181)
2023-12-01T09:04:35.176777+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1572)
2023-12-01T09:04:35.176804+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1529)
2023-12-01T09:04:35.176840+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1438)
2023-12-01T09:04:35.177447+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1181)
2023-12-01T09:04:35.177490+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1572)
2023-12-01T09:04:35.177518+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1529)
2023-12-01T09:04:35.177543+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1438)
2023-12-01T09:04:35.177569+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1181)
2023-12-01T09:04:35.177604+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1572)
2023-12-01T09:04:35.177875+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1529)
2023-12-01T09:04:35.178084+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1438)
2023-12-01T09:04:35.178312+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1181)
2023-12-01T09:04:35.178539+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at java.base/java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:350)
2023-12-01T09:04:35.178774+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.springframework.core.serializer.DefaultSerializer.serialize(DefaultSerializer.java:46)
2023-12-01T09:04:35.178999+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.springframework.core.serializer.Serializer.serializeToByteArray(Serializer.java:56)
2023-12-01T09:04:35.179226+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011at org.springframework.core.serializer.support.SerializingConverter.convert(SerializingConverter.java:60)
2023-12-01T09:04:35.179438+00:00 bankid-idp-app2 sunet-bankidp[122217]: #033[36mapp_1  |#033[0m #011... 138 common frames omitted
```

--- a/bankid-idp/env/local/developer.yml
+++ b/bankid-idp/env/local/developer.yml
@@ -74,7 +74,7 @@ bankid:
     user-message-defaults:
       fallback-sign-text:
         text: "Jag skriver härmed under den information som visades på föregående sida."
-        format: plain-text
+        format: plain_text
       login-text:
         text: "*Tänk på!* Logga aldrig in med ditt BankID då någon ringer och ber dig logga in."
         format: simple-markdown-v1

--- a/bankid-idp/src/test/resources/application.yml
+++ b/bankid-idp/src/test/resources/application.yml
@@ -22,7 +22,7 @@ bankid:
     user-message-defaults:
       fallback-sign-text:
         text: "Jag skriver härmed under den information som visades på föregående sida."
-        format: plain-text
+        format: plain_text
       login-text:
         text: "*Tänk på!* Logga aldrig in med ditt BankID då någon ringer och ber dig logga in."
         format: simple-markdown-v1

--- a/docs/configuration.md
+++ b/docs/configuration.md
@@ -144,7 +144,7 @@ bankid:
     user-message-defaults:
       fallback-sign-text:
         text: "I hereby sign the text that was displayed on the previous page."
-        format: plain-text
+        format: plain_text
       login-text:
         text: "*Note!*\n\nNever login using your BankID when someone is asking you to do this over the phone"
         format: simple-markdown-v1

--- a/samples/external-frontend/bankid-config/externalized-frontend.yml
+++ b/samples/external-frontend/bankid-config/externalized-frontend.yml
@@ -60,7 +60,7 @@ bankid:
     user-message-defaults:
       fallback-sign-text:
         text: "Jag skriver härmed under den information som visades på föregående sida."
-        format: plain-text
+        format: plain_text
       login-text:
         text: "*Tänk på!* Logga aldrig in med ditt BankID då någon ringer och ber dig logga in."
         format: simple-markdown-v1

--- a/samples/sandbox/config/sandbox.yml
+++ b/samples/sandbox/config/sandbox.yml
@@ -51,7 +51,7 @@ bankid:
     user-message-defaults:
       fallback-sign-text:
         text: "Jag skriver härmed under den information som visades på föregående sida."
-        format: plain-text
+        format: plain_text
       login-text:
         text: "*Tänk på!* Logga aldrig in med ditt BankID då någon ringer och ber dig logga in."
         format: simple-markdown-v1

